instance/migrations/0053_remove_subdomain_basedomain.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.5 on 2016-06-03 08:18
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('instance', '0052_add_internal_domain_fiels'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='openedxinstance',
+            name='internal_lms_domain',
+            field=models.CharField(max_length=100, unique=True),
+        ),
+        migrations.AlterField(
+            model_name='openedxinstance',
+            name='internal_lms_preview_domain',
+            field=models.CharField(max_length=100, unique=True),
+        ),
+        migrations.AlterField(
+            model_name='openedxinstance',
+            name='internal_studio_domain',
+            field=models.CharField(max_length=100, unique=True),
+        ),
```

Comment:
@itsjeyd I don't think I can make these columns unique at creation time, because I also want them to be non-nullable and at creation time when I start with empty columns I can't have both (non-null and unique). I have to make them unique only after I have filled them with unique values.